### DISCUSSION TITLE
学校机房电脑说 PCL2仓库 这个网页有危险内容

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3118] [提问] [answered]
红屏大字贴脸了，但是https没有划掉（浏览器是edge）
家用电脑没事
真的是学校机房防护等级高吗（我们老师也没设置啥相关内容）

**Comment by MoYuan-CN** [comment]
可能罢。也有可能机房电脑太旧了，根证书过期了之类的

**Reply by iPodtouch6557** [reply]
我们学校建校才三年，不可能老旧啊

**Reply by iPodtouch6557** [reply]
我想了下，好像根据之前测的我对某个网站的访问情况对比来看，确实是防护等级的问题
但是先挂在这吧

**Reply by MoYuan-CN** [reply]
三年... 如果都不更新的话...

**Reply by iPodtouch6557** [reply]
不可能，我的电脑11月刚装，此前我信息课没电脑(((
而且，我们学校应当不会不翻新这个，以我的了解来看

**Comment by WForst-Breeze** [comment]
我自己家的电脑也有这种现象哈哈哈哈，手动在地址栏改到别的页面就好了（比如访问 PCL 仓库主页面被扔红屏就改地址到 issues 页面就能访问了）🤔

**Comment by WorldHim** [comment]
github 是不可能有问题的。
试试继续访问看看会不会跳转到其他网页

**Comment by LTCatt** [comment, marked as answer]
似乎 PCL 的仓库被恶意举报了，只有 PCL 的仓库用 Edge 打开会报危险……
可以的话大家可以在微软交个反馈 Orz
### Edge 的红屏页面的详情里就有提交误报的选项

**Reply by iPodtouch6557** [reply]
交过了

**Comment by kong-que-dnf123** [comment]
我甚至edge打不开PCL的discussion类网页......
被迫换的google来看PCL的github
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/c4439247-973c-4caf-ac9d-13e8cfa48c2c)

**Comment by allMagicNB** [comment]
> 似乎 PCL 的仓库被恶意举报了，只有 PCL 的仓库用 Edge 打开会报危险……
> 可以的话大家可以在微软交个反馈 Orz

确实是被恶意举报了，但是仅限于[主仓库](https://github.com/Hex-Dragon/PCL2)
（ https://github.com/Hex-Dragon/PCL2 ）
页面会报告危险，无论Edge的哪个客户端……
![Screenshot_20240111_132605_com.microsoft.emmx.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/4d969a20-575c-4b0f-9e7e-5977822d0bf2)

**Comment by zkitefly** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/64117916/aaca8654-5951-487d-ab40-0480ca1925c8)

**Comment by HomoobyDX** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/120797993/1c198415-bed5-41a2-8657-73165ca80349)
直接访问cang'ku仓库主页不行，但是访问讨论页面能正常没报毒

**Comment by iPodtouch6557** [comment]
家用电脑也出现了问题，但是我几乎不从repo主页访问PCL2导致没发现，是被恶意举报了
没关系，我看到这个问题就交了误报反馈

**Comment by SuoHouDaiShi** [comment]
## 注意看页面标题“受到举报的不安全页面”，极有可能是被人恶意举报的，并且访问 Github Docs 或 Github 托管的其它仓库就没有问题……咳。
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/43d60c67-8f17-48fa-a420-11ab87f6fbba)
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/5cc9dd60-05ac-4347-97a9-003a3eb16651)
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/c6718210-da00-40f6-be1f-bf71f019cbb6)
---
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/669fb6c4-dea1-42ee-8dd2-35af36ef13c3)
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/828ebf0a-cd1a-480a-8059-244d0d5b3c1d)
已经把反馈丢给巨硬了，看巨硬咋说吧……
